data/yaml/venues.yaml

```diff
@@ -816,6 +816,9 @@ privatenlp:
 pylo:
   acronym: PYLO
   name: Workshop on Computational Modeling of Polysynthetic Languages
+quasy:
```

Comment:
@mjpost shall we still keep quasy as a venue here?

Reply:
Yes—it is a venue, right? And now it’s linked with its pre-2020 instances.